File: 5/Main.java
class Main {
  public static void main(String[] args) {
    int ans = 0;
    boolean notFound = true;
    for (int i=20; notFound;i+=20) {
      ans = i;
      for (int j=1; j<=20; j++) {
        if (i%j !=0) {
          notFound = true;
          break;
        } else {
          notFound = false;
        }
      }
    }
    System.out.println(ans);
  }
}

File: 5/Main.go
package main

import "fmt"

func main() {
	ans := 0
	notFound := true
	for i := 20; notFound; i += 20 {
		ans = i
		for j := 1; j <= 20; j++ {
			if i%j != 0 {
				notFound = true
				break
			} else {
				notFound = false
			}
		}
	}
	fmt.Println(ans)
}
